### PR TITLE
docs: network pages update

### PR DESCRIPTION
**Pull request by rube-de**
- Update emerald networks to table
- Add cipher network information page
- Change links in web3gateway

### CODE REVIEW

File: docs/dapp/cipher/network.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+id: network
+title: Network Information
```
Comment (aefhm) [inline docs/dapp/cipher/network.mdx]: (non-blocking) Our conventions appears to derive this header 1?

@matevz One of our [plugins](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs) auto generates this from the Markdown right?

File: docs/dapp/cipher/network.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+description: Network Information for Emerald
```
Comment (aefhm) [inline docs/dapp/cipher/network.mdx]:
```suggestion
description: Network Information for Cipher
```

**Review by aefhm — APPROVED** [review]

File: docs/dapp/cipher/network.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+| Name/Provider               | Mainnet URL                               | Testnet URL                               | EIP-3091 compatible |
+|-----------------------------|-------------------------------------------|-------------------------------------------|---------------------|
+| Oasis Scan ([Bit Cat])                      | [https://www.oasisscan.com/paratimes/000…7cb](https://www.oasisscan.com/paratimes/000000000000000000000000000000000000000000000000e199119c992377cb) | [https://testnet.oasisscan.com/paratimes/000…000](https://testnet.oasisscan.com/paratimes/0000000000000000000000000000000000000000000000000000000000000000) | No |
+
+[Bit Cat]: https://www.bitcat365.com/
+
+:::tip
+
+Only rudimentary block explorer features exist for Cipher. Consider debugging
+the Cipher transactions using the [`oasis paratime show`] command part of the
```
Comment (aefhm) [inline docs/dapp/cipher/network.mdx]:
```suggestion
Cipher transactions with the [`oasis paratime show`] command using the [Oasis CLI].
```

File: docs/dapp/cipher/network.mdx (hunk 1 above)
Comment (rube-de) [reply docs/dapp/cipher/network.mdx]: I removed all except description, as also discussed here https://github.com/oasisprotocol/sapphire-paratime/pull/415

**Pull request merged**